upgrading/automated_upgrades.adoc

```diff
@@ -247,6 +248,10 @@ you used in previous versions.
 You must not remove the `openshift_kubelet_name_override` parameter from your
 inventory file after you upgrade.
 ====
+.. If you manage *_/etc/origin/master/htpasswd_* manually, set
```

Comment:
I'd say:

If you manually manage the cluster's  \*_/etc/origin/master/htpasswd\_* file, add
\`openshift_master_manage_htpasswd=false\` to your inventory file to prevent the
upgrade process from overwriting the \*_htpasswd\_* file.

Reply:
I like that, thanks!